src/graphql/type/definition.py

```diff
@@ -554,8 +554,9 @@ def to_kwargs(self) -> GraphQLFieldKwargs:
     def __copy__(self) -> GraphQLField:  # pragma: no cover
         return self.__class__(**self.to_kwargs())
 
+TContext = TypeVar("TContext")
 
-class GraphQLResolveInfo(NamedTuple):
+class GraphQLResolveInfo(Generic[TContext], NamedTuple):
```

Comment:
```suggestion
class GraphQLResolveInfo(NamedTuple, Generic[TContext]):
```

I think it's nicer to always put the Generics at the end.

Reply:
I also noticed that multiple inheritance with NamedTuple is not supported with Python < 3.11 (see [here](https://github.com/python/cpython/issues/88089)). Since GraphQL-core supports Python > 3.6, we need to make a case distinction or catch the type error for older Python versions, and use a normal NamedTuple in that case.